plugin/nns/README.md

```diff
@@ -6,14 +6,14 @@
 
 ## Description
 
-The nns plugin try to get value from TXT records in provided neo node 
+The nns plugin tries to get value from TXT records in provided neo node 
 (lookup in [NNS smart contract](https://docs.neo.org/docs/en-us/reference/nns.html)).
 You can specify NNS domain to map DNS domain from request (default no mapping).
 
 ## Syntax
 
 ``` txt
-nns NEO_N3_CHAIN_ENDPOINT [NNS_DOMAIN]
+nns NEO_N3_CHAIN_ENDPOINT CONTRACT_ID [NNS_DOMAIN]
```

Comment:
Should we make the `CONTRACT_ID` optional?

Reply:
It's not convenient to have multiple independent optional  parameters. 
We decided to make `contract_address` required, but if we need contract with id=1 in sidechain, we just place `-` instead of address.